src/components/victory-scatter.jsx

```diff
@@ -405,6 +391,24 @@ class VScatter extends React.Component {
   }
 
   render() {
+    // If animating, return a `VictoryAnimation` element that will create
+    // a new `VictoryScatter` with nearly identical props, except (1) tweened
+    // and (2) `animate` set to null so we don't recurse forever.
+    if (this.props.animate) {
+      // Do less work by having `VictoryAnimation` tween only values that
+      // make sense to tween. In the future, allow customization of animated
+      // prop whitelist/blacklist?
+      const animateData = _.omit(this.props, [
+        "animate", "scale", "showLabels", "standalone"
+      ]);
+
+      return (
+        <VictoryAnimation {...this.props.animate} data={animateData}>
+          {props => <VictoryScatter {...this.props} {...props} animate={null}/>}
+        </VictoryAnimation>
+      );
+    }
+
     if (this.props.standalone === true) {
       return (
         <svg style={this.style.parent}>{this.plotDataPoints()}</svg>
```

Comment:
The rendering here seems a bit different than other components: if `standalone`, this renders an `<svg>` without a containing `<g>` that gets `style.parent`, but in VictoryChart for instance, there's a `<g>` with `style.parent` either way (the only difference is whether an `<svg>` wraps it or not.